core/src/main/java/io/questdb/cairo/SymbolMapReader.java

```diff
@@ -35,4 +35,6 @@
     boolean isDeleted();
 
     void updateSymbolCount(int count);
+
+    boolean containsNullValue();
```

Comment:
Can we pull this value up `StaticSymbolTable` interface level? The idea of static symbol table is that it is bounded, e.g. knows of its size and presumably that it knows that it has null?

Reply:
sure

Reply:
👍